addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -739,12 +747,54 @@ QUnit.test('use a command', async function (assert) {
     );
 });
 
+QUnit.test('no command should be displayed for this channel type', async function (assert) {
```

Comment:
_"chat channels should not have channel commands"_

Reply:
The current name of test is obscure, notably the part _"for this channel type"_. This forces us to read the test to check which channel type it's about (`'chat'`). Suggested name makes it apparent that it's about `'chat'` channels.

I also clarify "command" to "channel command" for the same reason we name the model `mail.channel_command` and not `mail.command` (command is too broad, and could be easily become misleading in understanding of code).

The use of plurals is just so the phrase is short and give a general expectation on all chat channels. This is minor suggestion, just personal opinion that the phrasing is more natural with plurals.

Reply:
Test is not about any specific channel type though. It's about having no channel type that matches the allowed type of the command.

Reply:
IMO, test name should indicate that there is no channel command precisely because of the channel type.
Otherwise, for the same reason you gave, this would force to read the test to know why there is no channel  command displayed.